src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1309,7 +1309,12 @@ public function registerForAutoconfiguration(string $interface)
     /**
      * Registers an attribute that will be used for autoconfiguring annotated classes.
      *
-     * The configurator will receive a ChildDefinition instance, an instance of the attribute and the corresponding \ReflectionClass, in that order.
+     * The configurator will receive (in this order):
```

Comment:
I'd suggest to add a psalm/phpstan signature for the configurator callable

Reply:
Applied your feedback.